app/components/media/media_card_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="sm:grid sm:grid-cols-12 sm:gap-x-6 pb-14 <%= classes %>">
+  <%= render(Media::ImageComponent.new(media: @media)) %>
+  <div class="sm:col-span-10 text-center sm:text-left space-y-3">
+    <%= render(Media::TitleComponent.new(media: @media)) %>
+    <%= render(Media::DescriptionComponent.new(media: @media)) %>
+  </div>
+</div>
```

Comment:
In current form, I don't think there's a lot of value in splitting the component up this way. There is the potential upside of being able to vary the CSS of the child components, but:

1. We're not currently doing that anywhere
2. There's no way with current design to actually pass those classes into the children to vary them

The downside is that a reasonably simple and easy to understand template with 14 lines with 1 component and 2 files is now split across 4 components and 8 files.

Ideally with this approach we would also prefer to use slots, but that's minor.

Unless we have a way to utilise the new structure and a use case for it, I dont think this extra abstraction is worth it. Happy to hear opposing thoughts